src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -465,6 +465,10 @@ public int getGeneralizedCost() {
     return generalizedCost;
   }
 
+  public int getGeneralizedCostIncludingPenalty() {
+    return generalizedCost - penaltyCost(accessPenalty) - penaltyCost(egressPenalty);
+  }
+
```

Comment:
I wonder why the name is  getGeneralizedCost`Including`Penalty, when the function actually excludes two penalty values?